### PR TITLE
[release] 20211103

### PR DESCRIPTION
**Pull request by jiqiang90**

### CODE REVIEW

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [0.15.0] - 2021-11-03
+### Changed
+- Update `create` function in model.ts template (#573)
```
Comment (ianhe8x) [inline packages/cli/CHANGELOG.md:13]:
```suggestion
- Provide a more proper ts typing for `static create()` function in model.ts template (#573)
```

File: packages/common/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [0.13.0] - 2021-11-03
+### Added
+- Add `abi` to to project CustomDataSourceBase (#547)
+- Add `address` and rule to project CustomDataSourceBase, improve Moonbeam EVM (#593);
+### Changed
+- Allow dictionary in ProjectManifest v0.2.0 (#578)
```
Comment (ianhe8x) [inline packages/common/CHANGELOG.md:16]: this can be moved to `Fixed`

File: packages/common/CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,14 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [0.13.0] - 2021-11-03
+### Added
+- Add `abi` to to project CustomDataSourceBase (#547)
```
Comment (ianhe8x) [inline packages/common/CHANGELOG.md:13]: Move `abi` and `address` under `processor.options` (#598)

File: packages/contract-processors/CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0] - 2021-11-03
+### Fixed
+- Fix evm tx from filed (#590)
```
Comment (ianhe8x) [inline packages/contract-processors/CHANGELOG.md:11]:
```suggestion
- Fix evm tx's `from` field (#590)
```

File: packages/cli/CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,14 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
...
+- Use jsonrpc client to fetch genesis hash , drop dependencies of polkadot-api (#595)
```
Comment (stwiname) [inline packages/cli/CHANGELOG.md:14]:
```suggestion
- Use jsonrpc client to fetch genesis hash , drop dependencies of polkadot-api (#595)
- Update `subql validate` to validate custom ds (#596)
```

File: packages/node/CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.24.0] - 2021-11-03
+### Added
+- Add moonbeam custom datasource processor (#547)
```
Comment (ianhe8x) [inline packages/node/CHANGELOG.md:11]:
```suggestion
- [custom ds] Read and feed assets to custom ds's `validate()` (#547)
```

File: packages/node/CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+### Changed
```
Comment (ianhe8x) [inline packages/node/CHANGELOG.md:12]: add `Improve error msg when fetch block fails (#602)`

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**